### PR TITLE
Try: Update the order of the section styles

### LINKED ISSUE #142: Add section styles
This is a **suggestion** for adding section styles, block style variations that can be applied to, in this case, the group block.

Each style would use an accessible combination of two or more colors from the palette, **and each style needs to continue to work when the user changes the color palette preset.** 

The purpose is to make it easy for users to change the style of a part of their content, pattern, or templates.

Section styles are .JSON files placed in the `styles` folder or a subfolder to `styles`.
Each file needs to include: schema, version, title, slug, one ore more block types, and the styles.

Section styles can not include settings. For example, it can set a font size, but not add new font size options to the typography control.

```
{
	"$schema": "https://schemas.wp.org/trunk/theme.json",
	"version": 3,
	"title": "Base",
	"slug": "section-1",
	"blockTypes": [ "core/group" ],
	"styles": {}
}
```

Each the section style would include:

- Background color and text color. Link color, if different from text color.
- Spacing
- Styles for elements:
  - Buttons
  - Headings

The CSS for these style variations are often repeated multiple times on the same page, which is why I recommend limiting the styles.

### PR DESCRIPTION
**Pull request by carolinan**
**Description**
I updated the order of the section styles from light to dark.
I placed the files in a new folder called sections.

I did not change the names, they are still called Style 1-5.

**Screenshots**
image

Before:
image

**Testing Instructions**

Add a group or columns block and confirm that the style sections have the correct color, but a different name and order than before the PR.
Add a few patterns that have a style section, and confirm that the color is the same as before the PR.

Contact location and link -yellow
Link in bio heading, paragraph, links and full-height image -purple
Hero podcast -pink
Logos -off white/ beige

### CODE REVIEW

File: styles/01-evening.json [hunk 1]
```diff
@@ -119,7 +119,7 @@
 			}
 		},
 		"variations": {
-			"section-1": {
```
Comment (juanfra) [inline styles/01-evening.json:122]: Do we want to order the sections by number in the different style variations?

**Review by juanfra — COMMENTED** [review]
Looking good and working well to me. I left a small comment (nit) about ordering the sections by number in the different style variations, like they were before. 

https://github.com/user-attachments/assets/b72cfc28-9f1a-42af-a14c-04ef76f03281

From my perspective, the order matches what @richtabor mentioned [here](https://github.com/WordPress/twentytwentyfive/issues/142#issuecomment-2378164800) (lightest to darkest), so that's good.

File: styles/01-evening.json (hunk 1 above)
Comment (carolinan) [reply styles/01-evening.json:122]: Oh I did not think of that. I will make an update in the afternoon.

Comment (carolinan) [reply styles/01-evening.json:122]: I did not find the time yesterday, trying it now.

**Comment by carolinan** [comment]
Evening:
The colors are still not sorted from light to dark but dark to light:
image

Morning: is also kind of OK
image

It gets worse from there:

Noon:
image

Dusk:
image

Afternoon:
image

Twilight:
image

Sunrise:
image

Midnight:
image

**Pull request merged**